Aviral Tyagi/problem3.hs

```diff
@@ -0,0 +1,7 @@
+elementAt :: [a] -> Int -> a
+elementAt (first : rest) 1 = first
+elementAt (_ : rest) n = elementAt rest (n-1)
+
```

Comment:
For the second implementation consider using a helper function. I have mentioned this in another review, https://github.com/p-paf/99problems/pull/3#discussion_r395966694.